ui/apps/platform/src/Containers/NetworkGraph/components/NetworkBreadcrumbs.tsx

```diff
@@ -14,10 +14,7 @@ type NetworkBreadcrumbsProps = {
 function NetworkBreadcrumbs({ clusters = [] }: NetworkBreadcrumbsProps) {
     const { searchFilter, setSearchFilter } = useURLSearch();
 
-    const { cluster: selectedClusterName } = getScopeHierarchy(searchFilter);
-
-    const selectedClusterId =
-        clusters.find((cluster) => cluster.name === selectedClusterName)?.id || '';
```

Comment:
I got extensive changes to this section in my currently open PR, to make it cleaner when adding support for deployments. In fact, it passes a minimum cluster object, just name and ID, after that change.

https://github.com/stackrox/stackrox/pull/4093/files#diff-3703b64f72da48d77b30d34395ab1993f2fd70e6d6c20e5585a999834ca19b2d